app/src/main/java/com/kickstarter/viewmodels/DiscoveryViewModel.java

```diff
@@ -65,8 +66,17 @@ public DiscoveryViewModel(final @NonNull Environment environment) {
     showProfile = profileClick;
     showSettings = settingsClick;
 
-    creatorDashboardButtonIsGone = environment.currentConfig().observable()
-      .map(config -> !coalesce(config.features().get(FeatureKey.ANDROID_CREATOR_VIEW), false));
+    final Observable<Boolean> userIsCreator = currentUser.observable()
+      .map(u -> u != null && u.createdProjectsCount() > 0);
+
+    final Observable<Boolean> userCanViewCreatorDash = environment.currentConfig().observable()
+      .map(config -> coalesce(config.features().get(FeatureKey.ANDROID_CREATOR_VIEW), false));
+
+    creatorDashboardButtonIsGone = Observable.combineLatest(
```

Comment:
i know the whole checkstyle thing is pending but `this.creatorDashboardButtonIsGone` 😁